src/test/java/org/jenkinsci/plugins/plaincredentials/BaseTest.java

```diff
@@ -56,7 +56,7 @@ public void secretTextBaseTest() throws IOException {
     public void secretFileBaseTest() throws IOException, URISyntaxException {
         DiskFileItem fileItem = createEmptyFileItem();
         
-        FileCredentialsImpl credential = new FileCredentialsImpl(CredentialsScope.GLOBAL, CRED_ID, "Test Secret file", fileItem, "keys.txt", Base64.encode(fileItem.get()).toString());
+        FileCredentialsImpl credential = new FileCredentialsImpl(CredentialsScope.GLOBAL, CRED_ID, "Test Secret file", fileItem, "keys.txt", SecretBytes.fromBytes(fileItem.get()));
```

Comment:
I recommend adding an explicit *extra* test using the deprecated constructor rather than retaining an existing usage so that it is clear to others that we explicitly want to verify the deprecated variant